docs/docsite/directive_desc.yml

```diff
@@ -0,0 +1,23 @@
+become: Boolean that controls if privilege escalation is used or not on Task execution.
+become_flags: A string of flag(s) to pass to the privilege escalation program when ``become`` is True.
+become_method: Which method of privilege escalation to use. i.e. sudo/su/etc.
+become_user: User that you 'become' after using privilege escalation, the remote/login user must have permissions to become this user.
```

Comment:
User that you 'become' after using privilege escalation on the remote system, the ``remote_user`` user must have permissions to become this user.